cmd2/argparse_custom.py

```diff
@@ -743,38 +744,46 @@ def _metavar_formatter(self, action, default_metavar) -> Callable:
             # End cmd2 customization
         else:
             result = default_metavar
+        return result
+
+    def _metavar_formatter(self, action, default_metavar) -> Callable:
```

Comment:
The name makes me think the callable returned here is the output formatter, but it's just trying to get the data into a standard data structure. Maybe call something like `_normalize_metavar()`?

Reply:
I can't rename this since it's an overrided `argparse` method.